### PR TITLE
Add 6.5.0 release

### PR DESCRIPTION
**Pull request by greeshmaswaminathan**
Add 6.5.0 release

### CODE REVIEW

File: data/en/get_started.json [hunk 1]
```diff
@@ -732,6 +889,57 @@
...
+                  {
+                    "buttonTitle": "CDAP 6.5.0 for Docker",
+                    "id": "version-650-mac",
+                    "buttonHref": "https://hub.docker.com/r/caskdata/cdap-sandbox/"
+                  }
+                ],
+                "dropdownItems": [
+                  {
+                    "active": true,
+                    "linkTitle": "CDAP 6.4.1 Docker",
```
Comment (avirmani) [inline data/en/get_started.json:908]: should be 6.5.0

File: data/en/get_started.json [hunk 2]
```diff
@@ -409,6 +488,56 @@
...
             },
+            {
+              "id": "vm-650",
+              "item": {
+                "title": "CDAP for VM",
+                "img": "/images/get-started/vm.svg",
+                "buttons": [
+                  {
+                    "buttonTitle": "CDAP 6.5.0 for VM",
+                    "id": "version-650-mac",
```
Comment (avirmani) [inline data/en/get_started.json:499]: should this id be version-650-vm?

File: data/en/get_started.json [hunk 3]
```diff
@@ -732,6 +889,57 @@
...
+            {
+              "id": "docker-650",
+              "item": {
+                "title": "CDAP for Docker",
+                "description": "Download a standalone zip file for trying out CDAP on your Linux or Mac OSX machine.",
+                "img": "/images/get-started/docker.png",
+                "buttons": [
+                  {
+                    "buttonTitle": "CDAP 6.5.0 for Docker",
+                    "id": "version-650-mac",
```
Comment (avirmani) [inline data/en/get_started.json:901]: should this id be version-650-docker?

**Review by avirmani — COMMENTED** [review]
Added some comments

File: data/en/get_started.json (hunk 1 above)
Comment (greeshmaswaminathan) [reply data/en/get_started.json]: Done

File: data/en/get_started.json (hunk 2 above)
Comment (greeshmaswaminathan) [reply data/en/get_started.json]: changed to be consistent.

File: data/en/get_started.json (hunk 3 above)
Comment (greeshmaswaminathan) [reply data/en/get_started.json]: changed to be consistent.

**Review by avirmani — APPROVED** [review]
LGTM

**Pull request merged**